web/crux/proto/crux.proto

```diff
@@ -83,6 +83,7 @@ service CruxDeployment {
   rpc GetDeploymentSecrets(DeploymentListSecretsRequest) returns (common.ListSecretsResponse);
   rpc CopyDeploymentSafe(IdRequest) returns (CreateEntityResponse);
   rpc CopyDeploymentUnsafe(IdRequest) returns (CreateEntityResponse);
+  rpc PreDeploymentCheck(IdRequest) returns (Empty);
```

Comment:
why do we need this?

Reply:
We need to check all required secrets to be filled before starting the deployment and I thought checking them on the backend would be better.